d/docling/docling_v2.36.0_ubi9.5.sh

```diff
@@ -173,6 +172,11 @@ cd $BUILD_HOME
 git clone https://github.com/pytorch/vision.git
 cd vision
 git checkout $TORCHVISION_VERSION
+
+# Below patch is needed to exclude the models that come under SWAG license (CC-BY-NC-4.0)
+wget https://raw.githubusercontent.com/ppc64le/build-scripts/refs/heads/master/t/torchvision/0001-Exclude-source-that-has-commercial-license.patch
```

Comment:
also wondering what happens when in the latest code if this file changes or renamed to something else will cause the build failure, rather we need to stick to sha based link if curl is used or refer the relative path if possible

may be @sumitd2 you can take care of this in a follow up PR